### DISCUSSION TITLE
Protocol: pre-execution errors with mutations / queries.

### DISCUSSION THREAD

**Discussion by nrbnlulu** [enisdenjo/graphql-ws#572] [Q&A] [answered]
The protocol states that `Error` message should be sent when
> This can occur before execution starts, usually due to validation errors, or during the execution of the request.

Does it applies for mutation / queries as well?
Because the protocol [also states that ](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md#single-result-operation)

> Server dispatches the result with the Next message

**Comment by enisdenjo** [comment, marked as answer]
Yes, it applies to any operation type. Validation happens on all requests.
